### ISSUE TITLE
[BUG] Can't reach port 8001

### ISSUE THREAD

**Issue by dasarne** [tobybatch/kimai2#214]
**Describe the bug**
Starting the docker-container does not open port 8001 but 9000

[root@jim ~]# nmap 172.17.0.6           
Starting Nmap 7.91 ( https://nmap.org ) at 2021-02-14 19:59 CET
Nmap scan report for 172.17.0.6
Host is up (0.0000050s latency).
Not shown: 999 closed ports
PORT     STATE SERVICE
9000/tcp open  cslistener
MAC Address: 02:42:AC:11:00:06 (Unknown)

Nmap done: 1 IP address (1 host up) scanned in 0.26 seconds

Port 8001 isn't open and on port 9000 there is no answer.
The same docker-call with  ":latest-dev" works like a charm.

**To Reproduce**
s.a.

**Desktop (please complete the following information):**
 - OS: Manjaro Linux q
 - Docker version: Docker version 20.10.3, build 48d30b5b32

**Command used to run the container**
docker run -ti -p 8001:8001 -e DATABASE_URL=mysql://foo:bar@my-server.de:3306/kimai2 -e APP_SECRET=xxxxxxx -e MAILER_FROM=kimai@my-server.de -e MAILER_URL="smtp://my-server.de:25?encryption=&auth_mode=" -e DB_TYPE=mysql -e TRUSTED_HOSTS=nginx,localhost,127.0.0.1,my-server.de,xxx.xxx.xxx.xxx --name kimai kimai/kimai2

**Comment by tobybatch** [comment]
Hi, You have started an FPM image without any web server to serve the site. Try this:

```
docker run -ti -p 8001:8001 \
    -e DATABASE_URL=mysql://foo:bar@my-server.de:3306/kimai2 \
    -e APP_SECRET=xxxxxxx \
    -e MAILER_FROM=kimai@my-server.de \
    -e MAILER_URL="smtp://my-server.de:25?encryption=&auth_mode=" \
    -e DB_TYPE=mysql \
    -e TRUSTED_HOSTS=nginx,localhost,127.0.0.1,my-server.de,xxx.xxx.xxx.xxx \
    --name kimai \
    kimai/kimai2:apache-debian-1.13-prod
```

**Comment by dasarne** [comment]
That fixes the problem! 
Thanks for the support. Maybe you should add this info to the documentation ;-)

But... 
- how does that update? 
- is there anyone who needs a docker container with php inside the doesn't serve anything?
- wouldn't it be a better to include the apache in the docker-image? This is then closer to the development version...

Yours 
 Arne

**Comment by tobybatch** [comment]
This is a little out of scope for this repo but there are two main ways of runnnig PHP web site. There is a fairly good (but old) comparison here https://razercattleya.wordpress.com/2013/10/25/mod_php-vs-fastcgi-vs-php-fpm-for-web-server-scripting/

TLDR;
mod_apache is easier to install and set up but is _way_ slower
php-fpm is harder to set up and requires a second server but scales better and is a lot quicker.
